src/components/ButtonGroup/ButtonGroup.scss

```diff
@@ -20,10 +20,12 @@
 
     // Manage separator
     .variant_default {
-        &:hover,
-        &:active,
-        &:focus {
-            border-left-color: currentColor !important;
+        .color_default {
+            &:hover,
+            &:active,
+            &:focus {
+                border-left-color: currentColor !important;
```

Comment:
Can't you add more specificity instead of using !important ? Maybe a &&:hover,&&:active,&&:focus ?

Reply:
Our stylelint config prevents this usage.

Anyway, the styling button has been updated here: [MOON-112: Design improvements](https://github.com/Jahia/moonstone/pull/133),  so we don't need this piece of code. I remove it

Reply:
stylelint prevents `&&` but allows `!important` ? a base rule would be in my opinion to just forbid any `!important` anywhere :-)

Reply:
would be good to add declaration-no-important to the config